ui/javascripts/app/helpers.js

```diff
@@ -91,3 +91,63 @@ function notify(message, ttl) {
   window.notifications = [];
   window.notifications.push(notification);
 }
+
+// Tomography
+
+Ember.Handlebars.helper('tomographyGraph', function(tomography, size) {
+
+  // This is ugly, but I'm working around bugs with Handlebars and templating
+  // parts of svgs. Basically things render correctly the first time, but when
+  // stuff is updated for subsequent go arounds the templated parts don't show.
+  // It appears (based on google searches) that the replaced elements aren't
+  // being interpreted as http://www.w3.org/2000/svg. Anyway, this works and
+  // if/when Handlebars fixes the underlying issues all of this can be cleaned
+  // up drastically.
+
+  var max = Math.max.apply(null, tomography.distances);
+  var insetSize = size / 2 - 8;
+  var buf = '' +
+'      <svg width="' + size + '" height="' + size + '">' +
+'        <g class="tomography" transform="translate(' + (size / 2) + ', ' + (size / 2) + ')">' +
+'          <g>' +
+'            <circle class="background" r="' + insetSize + '"/>' +
+'            <circle class="axis" r="' + (insetSize * 0.25) + '"/>' +
+'            <circle class="axis" r="' + (insetSize * 0.5) + '"/>' +
+'            <circle class="axis" r="' + (insetSize * 0.75) + '"/>' +
+'            <circle class="border" r="' + insetSize + '"/>' +
+'          </g>' +
+'          <g class="lines">';
+  var sampling = 360 / tomography.n;
+  distances = tomography.distances.filter(function () {
+    return Math.random() < sampling
```

Comment:
It would be kind of cool to always include the sample if the index is 0 or `n-1`, that way the min and max samples are always included in the plot.